common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -45,6 +45,16 @@ class SequenceFields(object):
         scope=Scope.user_state,
     )
 
+    # Entrance Exam flag -- see cms/contentstore/views/entrance_exam.py for usage
+    is_entrance_exam = Boolean(
+        display_name=_("Is Entrance Exam"),
+        help=_(
+            "Tag this course module as an Entrance Exam.  " +
+            "Note, you must enable Entrance Exams for this course setting to take effect."
+        ),
+        scope=Scope.settings,
+    )
+
```

Comment:
Happy to employ inherited metadata -- let's discuss how to go about doing so (an example of it in action would be fantastic)